src/adapters/methods/vision.py

```diff
@@ -0,0 +1,66 @@
+# Module to support adapter training for vision related tasks
```

Comment:
I believe it's fully fine to just use the implementation of torchvision here if there's no functional difference. We most likely require that package anyway for training vision models (e.g. for image transforms, see [here](https://github.com/huggingface/transformers/blob/1d063793318b20654ebb850f48f43e0a247ab7bb/examples/pytorch/image-classification/run_image_classification.py#L27)).

Reply:
Sounds good! then i will import the `stochastic_depth` function from torchvision inside `src/adapters/methods/modeling.py`